runtime/autoload/provider/ruby.vim

```diff
@@ -47,8 +43,24 @@ function! provider#ruby#Call(method, args) abort
   return call('rpcrequest', insert(insert(a:args, 'ruby_'.a:method), s:host))
 endfunction
 
+function! s:detect()
+  if exists("g:ruby_host_prog")
+    return g:ruby_host_prog
+  elseif has('win32')
+    return exepath('neovim-ruby-host.bat')
+  else
+    let p = exepath('neovim-ruby-host')
+    if empty(p)
```

Comment:
Keep the ternary so that `empty(p)` works for both shims.

Reply:
Hmm, not sure what you mean. `exepath()` will return an empty string anyway when there's no executable found. And this way we only use the `system()` check for non-Win systems.

Reply:
`rbenv` doesn't work on Windows?

Reply:
Well, at least not the hugely popular [rbenv/rbenv](https://github.com/rbenv/rbenv).

I found [nak1114/rbenv-win](https://github.com/nak1114/rbenv-win), but it's a complete rewrite and I never heard someone using rbenv on Windows. I'm not even sure what `v:shell_error` gets set to on Windows.